### DISCUSSION TITLE
Is workflow-core compatible with .net 6?

### DISCUSSION THREAD

**Discussion by vigneshwarb6** [danielgerlag/workflow-core#1104] [Q&A]
Is workflow-core compatible with .net 6?, we have been using workflow-core in our project which is running on .net core 3.1 given .net core 3.1 is getting decomissioned in Dec 2022 , we are planning to migrate to .net 6 and want to check here if workflow-core is compatible with .net 6 .

**Comment by IvanJosipovic** [comment]
Yes, I have been using it in production with .Net 6 for ages.

**Reply by contactsureshh** [reply]
Hi, what stable version of Workflow-core is compatible with .NET6 ?
